lib/generator.js

```diff
@@ -2026,6 +2024,7 @@
             
     exports.Generator         = Generator;
     exports.createGenerator   = createGenerator;
+    exports.logStream         = _logStream;
```

Comment:
This seems mildly janky since the two following lines use and underscore in the opposite direction... but I can't convince myself this is wrong...at least without more structural changes.  Open to suggestions.